seyik/app.js

```diff
@@ -111,6 +110,57 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/modify", async (req, res) => {
```

Comment:
```suggestion
app.patch("/posts", async (req, res) => {
```
http method인 `PATCH` 자체가 수정을 의미하기 때문에, 엔드포인트(목적 자원)에 `modify`라고 작성하는 것 보다는, 어떤걸 PATCH 할 지 써주는게 보다 직관적입니다. `/posts`처럼요. 이와 같이 작성하면 http method와 엔드포인트의 조합만으로 무엇을(`what`), 어떻게(`how`) 하겠다는 것을 자명하게 나타낼 수 있습니다. 이러한 엔드포인트 설계 아키텍쳐를 REST라고 하는데 이는 금주에 세션에서 자세한 내용과 함께, 다시 한 번 다룰 예정이니 지금은 '그렇구나~' 인지만 하고 넘어가시면 될 것 같습니다. 수정해주세요!